### PR TITLE
Add longrunning API from gapic generated code

### PR DESCRIPTION
**Pull request by swcloud**

### CODE REVIEW

File: lib/google/longrunning/operations_api.rb [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+# For the short term, the refresh process will only be runnable by Google
+# engineers.
+#
+# The only allowed edits are to method and file documentation. A 3-way
+# merge preserves those additions if the generated source changes.
+
+require "json"
+require "pathname"
+
+require "google/gax"
```
Comment (jmuk) [inline lib/google/longrunning/operations_api.rb]: Be careful about the circular dependency; when the final design of LRO is implemented, some part of gax library will depend on this file. cc @landrito

File: lib/google/longrunning/operations_pb.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```
Comment (jmuk) [inline lib/google/longrunning/operations_pb.rb]: I think we've decided these pb.rb files are either in a part of googleapis-common-protos package or an independent package.

File: lib/google/gax/version.rb [hunk 3]
```diff
@@ -29,6 +29,6 @@
 
 module Google
   module Gax
-    VERSION = '0.5.1'.freeze
+    VERSION = '0.5.2'.freeze
```
Comment (jmuk) [inline lib/google/gax/version.rb]: This type of the change would require an increment on the minor version part, not the patch part (see http://semver.org/).

Comment (swcloud) [inline lib/google/gax/version.rb]: ok. I'll change it to '0.6.0'.

File: lib/google/longrunning/operations_pb.rb (hunk 2 above)
Comment (swcloud) [inline lib/google/longrunning/operations_pb.rb]: will remove _pb.rb files

**Comment by jmuk** [comment]
Oh, and you'd have to add dependency on `googleapis-common-protos` package.

**Comment by swcloud** [comment]
Added the dependency and fixed two rubocop offenses.

**Pull request merged**